sensing/pointcloud_preprocessor/src/outlier_filter/ring_outlier_filter_nodelet.cpp

```diff
@@ -33,8 +33,7 @@ RingOutlierFilterComponent::RingOutlierFilterComponent(const rclcpp::NodeOptions
     using tier4_autoware_utils::StopWatch;
     stop_watch_ptr_ = std::make_unique<StopWatch<std::chrono::milliseconds>>();
     debug_publisher_ = std::make_unique<DebugPublisher>(this, "ring_outlier_filter");
-    excluded_points_publisher_ =
-      this->create_publisher<sensor_msgs::msg::PointCloud2>("debug/ring_outlier_filter", 1);
+    noise_points_publisher_ = this->create_publisher<PointCloud2>("noise/ring_outlier_filter", 1);
```

Comment:
Do you plan to use it formally as a process flow? For debugging?

Reply:
As I explained in the README, this topic will be published for debug reason.
I don't want to increase processing time.
But in the future implement, if it's useful information to calculate visibility score, I would like to use it in the node or outside of the node.
In any case, I suggest having option to visualize the information.
What do you think?

Reply:
@kyoichi-sugahara thanks for reply :+1: 
If it is for debugging, how about the following? The naming is unified with debug in other nodes.
```suggestion
    noise_points_publisher_ = this->create_publisher<PointCloud2>("debug/outlier_pointcloud", 1);
```

Reply:
change from `noise/ring_outlier_filter` to `debug/ring_outlier_filter` in https://github.com/autowarefoundation/autoware.universe/pull/6581/commits/77c0ab44c2d38a751d71764c5ba7ad5da632ffff to distinguish from dual_return_outlier_filter.
Is that ok?

Reply:
:+1: